src/otx/algo/visual_prompting/zero_shot_segment_anything.py

```diff
@@ -841,29 +843,52 @@ def initialize_reference_info(self) -> None:
         self.register_buffer("reference_feats", torch.zeros(0, 1, self.model.embed_dim), False)
         self.register_buffer("used_indices", torch.tensor([], dtype=torch.int64), False)
 
-    def _find_latest_reference_info(self, root: Path) -> str | None:
-        """Find latest reference info to be used."""
-        if not Path.is_dir(root):
-            return None
-        if len(stamps := sorted(os.listdir(root), reverse=True)) > 0:
-            return stamps[0]
-        return None
+    def save_reference_info(self, default_root_dir: Path | str) -> None:
+        """Save reference info."""
+        reference_info = {
+            "reference_feats": self.reference_feats,
+            "used_indices": self.used_indices,
+        }
```

Comment:
I'm curious why these tensors cannot be not included in the model checkpoint.

Reply:
We changed direction to return those tensors, not including them in the model.
But for OTX CLI standalone usecase, they are required for inference, so I and @wonjuleee decided to save them at the specific location and load them.

Reply:
If you are worrying about the big size of checkpoint file, I think you can still amortize this to the model checkpoint. You can override `load_state_dict()` and `state_dict()` by dropping the model weight part from them. This will make it more streamlinedwith the checkpoint callback.